src/chainer.test.js

```diff
@@ -1 +1,50 @@
-const { chainer } = require('./chainer');
+'use strict';
+
+describe('chainer', () => {
+  const { chainer } = require('./chainer');
+
+  it(`should correctly chain functions and return the expected result`, () => {
+    const addOne = (x) => x + 1;
```

Comment:
The same set of functions (addOne, multiplyByTwo, subtractThree) is declared in each test case which is repetitive. These functions should be declared once before all test cases.